Build-qt.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
```

Comment:
Never-mind, since the script handle both MacOSx and Linux, you can ignore.

Sorry for the noise